_rules/error-field-is-not-identified-using-color-only

```diff
@@ -0,0 +1,107 @@
+---
+id:
+name: Error field is not identified using color differences only
+rule_type: atomic
+description: |
+  This rule checks that error fields can be identified not only by color differences but through another visual means.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  -  DOM Tree
+  -  CSS Styling
+acknowledgments:
+  authors:
+    -  Giacomo Petri
+---
+
+## Applicability
+
+This rule applies to any error field that is [visible][].
+
+## Expectation (1)
+
+Each test target, together with its visual context, identify the presence of errors not only through color differences.
+
+## Assumptions
+
+If the presence of errors is conveyed through the use of colors that differ not only in their hue, but that also have a difference in lightness and the difference in relative luminance between the colors leads to a contrast ratio of 3:1 or greater, then this counts as an additional visual mean.
+
+*Note*: It is possible for an error field to be distinguishable from adjacent fields but still not have a meaningful error message. In that case, it would pass Success Criterion 1.4.1: Use of Color but still fail [Success Criterion 3.3.1: Error Identification](https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html) and, if applicable, [Scuccess Criterion 3.3.3: Error Suggestion](https://www.w3.org/WAI/WCAG21/Understanding/error-suggestion.html).
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+- [Understanding Success Criterion 1.4.1: Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)
+- [F81 - Failure of Success Criterion 1.4.1 due to identifying required or error fields using color differences only](https://www.w3.org/WAI/WCAG21/Techniques/failures/F81.html)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
```

Comment:
I feel we need examples that use something else than text inputs and other errors than absence of a required field. That can be both other built-in errors (check the list of [constraint validation errors](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#validity-states) for inspiration); or custom errors (e.g. "password must contain at least 8 characters, …" that is usually checked by some script).

Notably, that would show that the rule also considers custom errors.
Also, I do believe that if built-in constraint validations are failing, the UA should automagically set `aria-invalid` (needs to be tested).

Reply:
Updated with different examples and added more details in what's failing.